ee/tuf/autoupdate_test.go

```diff
@@ -75,6 +75,11 @@ func TestExecute_launcherUpdate(t *testing.T) {
 	testReleaseVersion := "1.2.3"
 	tufServerUrl, rootJson := tufci.InitRemoteTufServer(t, testReleaseVersion)
 	s := setupStorage(t)
+	// setup fake osqueryd binary to mock file existence for currentRunningVersion
+	fakeOsqBinaryPath := filepath.Join(testRootDir, "osqueryd")
```

Comment:
Think you will want this to be `osqueryd.exe` for running tests on Windows. I think there's a utility func for this, hang on...

Reply:
`executableLocation`?

Reply:
👍 will update, ty!